src/common.rs

```diff
@@ -233,6 +234,26 @@ impl Config {
         self.target_rustcflags = Some(flags);
     }
 
+    /// Remove rmeta files from target `deps` directory
```

Comment:
I'd like a little more documentation here, to explain why this is useful. IIRC this feature is useful when working with `cargo check` vs `cargo build` because the two methods don't share any information.